web/locales/cs.yml

```diff
@@ -1,4 +1,5 @@
 cs:
+  "Recurring Jobs": Pravidelně opakované
```

Comment:
This was the only way I could get the tab title to work when using the locale switcher at the bottom of Sidekiq web UI. 🤷‍♂️